packages/perseus/src/widgets/interactive-graphs/types.ts

```diff
@@ -21,7 +21,7 @@ export type MafsGraphProps<T extends InteractiveGraphState> = {
 // end up in different sections of the DOM.
 export type InteractiveGraphElementSuite = {
     graph: ReactNode;
-    screenreaderDescription: ReactNode;
+    interactiveElementsDescription: ReactNode;
```

Comment:
~~I'm curious about why this is a ReactNode rather than a string, which is what `describePointGraph` returns in point.tsx?~~

Is this a ReactNode so that you can use the `usePerseusI18n` hook? (Maybe a comment would be helpful for confused individuals like me 😅)